app/Handlers/Events/SendIncidentEmailNotificationHandler.php

```diff
@@ -69,6 +69,7 @@ public function handle(IncidentHasReportedEvent $event)
                 $mail = [
                     'email'           => $subscriber->email,
                     'subject'         => 'New incident reported.',
+                    'incident'        => $event->incident,
```

Comment:
BTW I'm getting this error `Trying to get property of non-object (View: /home/vagrant/code/cachet/resources/views/emails/incidents/new-html.blade.php)` exactly on the if $incident->component